terraform/modules/self-service/modules/cognito/cognito.tf

```diff
@@ -71,16 +168,13 @@ resource "aws_cognito_user_pool" "user_pool" {
       "mfa_configuration"
     ]
 
-    prevent_destroy = true
+    prevent_destroy = false
```

Comment:
Is this because you intend to destroy it?

Reply:
We'll need to destroy to allow the MFA configuration to go through.